content/en/integrations/guide/mongo-custom-query-collection.md

```diff
@@ -0,0 +1,158 @@
+
+{{< tabs >}}
+{{% tab "Count" %}}
+
+To monitor how many users are active at a given time for example. Your [Mongo count command][1] would be:
+
+```
+db.runCommand( {count: user_collection, query: {active:true}})
+```
+
+Which would correspond to the following `custom_queries` YAML configuration inside your `mongo.d/conf.yaml` file:
+
+```
+custom_queries:
+  - metric_prefix: mongo.users
+    query: {"count": "user_collection", "query": {"active":"true"}}
+    count_type: gauge
+    tags:
+      - user:active
+```
+
+This would emit one `gauge` metric `mongo.users` with one tag: `user:active`.
+
+**Note**: The metric type defined is `gauge`. See the [metric type documentation][2] to learn more.
+
+
+[1]: https://docs.mongodb.com/manual/reference/command/count/#dbcmd.count
+[2]: /developers/metrics/#metric-types
+{{% /tab %}}
+{{% tab "Find" %}}
+
+To monitor what's the average age per user, your [Mongo find command][1] would be:
+
+```
+db.runCommand( {find: user_colleciton, filter: {active:true} )
+```
+
+Which would correspond to the following `custom_queries` YAML configuration inside your `mongo.d/conf.yaml` file:
+
+```
+custom_queries:
+  - metric_prefix: mongo.example2
+    query: {"find": "user_colleciton", "filter": {"active":"true"}}
+    fields:
+      - field_name: name
+        name: name
+        type: tag
+      - field_name: age
+        name: user.age
+        type: gauge
+
+```
+
+This would emit one `gauge` metric `mongo.example2.user.age` with two tags: `name:foo` and `name:foobar`
+
+**Note**: The metric type defined is `gauge`. See the [metric type documentation][2] to learn more.
+
+[1]: https://docs.mongodb.com/manual/reference/command/find/#dbcmd.find
+[2]: /developers/metrics/#metric-types
+{{% /tab %}}
+{{% tab "Aggregate" %}}
+
+To monitor what's the average age for admin and non admin user, your [Mongo aggregate command][1] would be:
+```
+db.runCommand(
+              {
+                'aggregate': "user_collection",
+                'pipeline': [
+                  {"$match": {"is_actif": "true"}},
+                  {"$group": {"_id": "$is_admin", "age_avg": {"$avg": "$age"}}}
+                ],
+                'cursor': {}
+              }
+            )
+```
+
+Which would correspond to the following `custom_queries` YAML configuration inside your `mongo.d/conf.yaml` file:
+
+```
+custom_queries:
+  - metric_prefix: mongo.example3
+    query: {"aggregate": "user_collection","pipeline": [{"$match": {"is_actif": "true"}},{"$group": {"_id": "$is_admin", "total": {"$avg": "$age"}}}],"cursor": {}}
+    fields:
+      - field_name: age_avg
+        name: user.age
+        type: gauge
+      - field_name: _id
+        name: is_admin
+        type: tag
+    tags:
+      - test:mongodb
+```
+
+This would emit one `gauge` metric `mongo.example3.user.age` with two tags: `is_admin:true` and `is_admin:true` representing the average age of users for each tags.
```

Comment:
```suggestion
This would emit one `gauge` metric `mongo.example3.user.age` with two tags: `is_admin:true` and `is_admin:false` representing the average age of users for each tags.
```